### PR TITLE
kind of nice solution for introducing welcome screen

### PR DESCRIPTION
**Pull request by vujevits**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -144,12 +144,33 @@ const AppNavigator = StackNavigator(Scenes,
...
+        screen: IdentitySettingsContainer,
+    },
+}, {
+    mode: 'card',
+    navigationOptions: {
+        header: null,
+    },
+});
+
+const SwNavigator = SwitchNavigator({
```
Comment (agazso) [inline src/App.tsx]: I think MainNavigator or InitialNavigator would be a better name for this

**Comment by agazso** [comment]
👍

**Comment by agazso** [comment]
One thing I found: there is a Back button on the Welcome screen, it would be great to remove it and put a title instead.

**Comment by vujevits** [comment]
Introduced a 2 page onboarding where on the second screen the username and the avatar can be selected

File: src/App.tsx (hunk 1 above)
Comment (vujevits) [reply src/App.tsx]: solved

**Pull request merged**